auslib/db.py

```diff
@@ -1970,8 +1970,8 @@ def getOptions(self, username, permission, transaction=None):
             raise ValueError('Permission "%s" doesn\'t exist' % permission)
 
     def getUserRoles(self, username, transaction=None):
-        res = self.user_roles.select(where=[self.user_roles.username == username], columns=[self.user_roles.role], distinct=True, transaction=transaction)
-        return [r["role"] for r in res]
+        res = self.user_roles.select(where=[self.user_roles.username == username], columns=[self.user_roles.role, self.user_roles.data_version], distinct=True, transaction=transaction)
+        return [{"role": r["role"], "data_version": r["data_version"]} for r in res]
```

Comment:
hi @mozbhearsum I require the roles to return the `data_version` because this is required when revoking a role from the ui